open_city_profile/settings.py

```diff
@@ -282,20 +283,24 @@
 # Month from which on the membership will last until the next year, instead of ending in the current year
 YOUTH_MEMBERSHIP_FULL_SEASON_START_MONTH = 5
 
+if not DEBUG:
+    audit_handler = {
+        "level": "INFO",
+        "class": "logging.FileHandler",
+        "filename": "./audit.log",
```

Comment:
I think this file will not currently survive when the container is restarted in qa/production environment. Since the logger itself will propagate the log message it probably also ends up in the normal log. I'm just wondering if the file logger is something that we really want here?

Reply:
Good catch. We want to log to stdout only.